README.md

```diff
@@ -8,10 +8,10 @@ A c library implementing the POST API setup method for general-purpose CPUs and
 
 ## Runtime System Requirements
 
-Windows 10, macOS or Ubuntu.
+Windows 10/11, macOS or Ubuntu.
```

Comment:
We really don't support other Linux distros? Why not? At least worth adding a comment here to this effect

Reply:
The problem is that we checked on Ubuntu "only", let me check on other linux distro.

Reply:
I think it's sufficient to just add a note here saying we've only tested and confirmed on Ubuntu (which version(s)?), and that it may work on others but we cannot confirm

Reply:
Just please note that it's build not usage.

Reply:
Will check. Generally all should work.